colossalai/nn/layer/moe/experts.py

```diff
@@ -34,7 +34,7 @@ class Experts(MoeExperts):
     :type num_experts: int
     """
 
-    def __init__(self, expert, num_experts, **expert_args):
+    def __init__(self, expert: nn.Module, num_experts: int, **expert_args):

```

Comment:
Here parameter `expert` is not a nn.Module instance. It is just a class name, the expert instance will be constructed later.

Reply:
OK, its a TYPE[nn.Module]

see https://docs.python.org/3/library/typing.html